pkg/server/server.go

```diff
@@ -41,162 +41,169 @@ import (
 	AlertManagerTenancyProxyConfig      *proxy.Config
-	MeteringProxyConfig                 *proxy.Config
-	TerminalProxyTLSConfig              *tls.Config
-	PluginsProxyTLSConfig               *tls.Config
-	GitOpsProxyConfig                   *proxy.Config
+	AlertManagerUserWorkloadHost        string
+	AlertManagerUserWorkloadProxyConfig *proxy.Config
+	Authers                             map[string]*auth.Authenticator
+	AuthMetrics                         *auth.Metrics // TODO: only needed because handleLogoutMulticluster is part of the server type
+	BaseURL                             *url.URL
+	Branding                            string
 	ClusterManagementProxyConfig        *proxy.Config
+	ControlPlaneTopology                string
+	CopiedCSVsDisabled                  map[string]bool
+	CustomLogoFile                      string
+	CustomProductName                   string
+	DevCatalogCategories                string
+	DevCatalogTypes                     string
+	DocumentationBaseURL                *url.URL
+	EnabledConsolePlugins               serverconfig.MultiKeyValue
+	GitOpsProxyConfig                   *proxy.Config
+	GOARCH                              string
+	GOOS                                string
+	GrafanaPublicURL                    *url.URL
+	HubConsoleURL                       *url.URL
+	I18nNamespaces                      []string
+	InactivityTimeout                   int
+	K8sClient                           *http.Client
+	K8sProxyConfig                      *proxy.Config
+	KnativeChannelCRDLister             ResourceLister
+	KnativeEventSourceCRDLister         ResourceLister
+	KubeAPIServerURL                    string
+	KubectlClientID                     string
+	KubeVersion                         string
+	LoadTestFactor                      int
+	LogoutRedirect                      *url.URL
 	ManagedClusterProxyConfig           *proxy.Config
-	// A lister for resource listing of a particular kind
-	MonitoringDashboardConfigMapLister ResourceLister
-	KnativeEventSourceCRDLister        ResourceLister
-	KnativeChannelCRDLister            ResourceLister
-	GOARCH                             string
-	GOOS                               string
-	// Monitoring and Logging related URLs
-	AlertManagerPublicURL        *url.URL
-	GrafanaPublicURL             *url.URL
-	PrometheusPublicURL          *url.URL
-	ThanosPublicURL              *url.URL
-	AlertManagerUserWorkloadHost string
-	AlertManagerTenancyHost      string
-	DevCatalogCategories         string
-	DevCatalogTypes              string
-	UserSettingsLocation         string
-	QuickStarts                  string
-	AddPage                      string
-	ProjectAccessClusterRoles    string
-	Perspectives                 string
-	Telemetry                    serverconfig.MultiKeyValue
-	CopiedCSVsDisabled           map[string]bool
-	HubConsoleURL                *url.URL
-
-	// TODO: only needed because handleLogoutMulticluster is part of the server type
-	AuthMetrics *auth.Metrics
```

Comment:
Everything above here is just housekeeping. Alphabetized struct properties and constants.